### DISCUSSION TITLE
Commit signing

### DISCUSSION THREAD

**Discussion by shivai** [fluxcd/flux2#3812] [Q&A]
I want to config Flux to sign its commit on the repository. I passed gpg key in bootstrap command but it just signs the first commit that it does for creating flux-system directory, but for example when I have a new tag in my image repository, Flux will change the image tag in my manifest in my repo and commit it then apply it in my Kubernetes cluster, now I want to sign this kind of commits as well.

**Comment by makkes** [comment]
https://fluxcd.io/flux/components/image/imageupdateautomations/#commit explains the signing options available to ImageUpdateAutomations. You'll have to configure your manifests accordingly since the bootstrapping doesn't create these.
